crates/stackable-telemetry/src/tracing/mod.rs

```diff
@@ -88,7 +88,43 @@ pub enum Error {
 /// }
 /// ```
 ///
-/// ## Basic configuration
+/// ## Pre-configured Tracing Instance
+///
+/// There are two different styles to configure a [`Tracing`] instance: Using an opinionated pre-
+/// configured instance or a fully customizable builder. The first option should be suited for
+/// pretty much all operators by using sane defaults and applying best practices out-of-the-box.
+/// [`Tracing::pre_configured`] lists details about environment variables, filter levels and
+/// defaults used.
```

Comment:
I think this section was supposed to move up above the level-2 headings 9as it describes both styles)?

Reply:
Now I'm not so sure, since there's another Level 2 heading for "Builder" below.

Reply:
This is fine as is